### PR TITLE
BugFix: Update ENV Values

### PR DESCRIPTION
**Pull request by MacDeluca**
## Links to Jira Tickets

- {Include a link to all applicable Jira tickets}

## Description of Changes

- {List all relevant code changes. Include any changes to the business workflow that might not be obvious to the reviewers of this PR.}

## Testing Notes

- {List any relevant testing considerations, necessary pre-reqs, and areas of the app to focus on. Specifically, include anything that will help the reviewers of this PR verify the code is functioning as expected.}

### CODE REVIEW

File: api/.pipeline/templates/api.dc.yaml [hunk 1]
```diff
@@ -336,6 +346,13 @@ objects:
                   value: ${KEYCLOAK_API_HOST}
                 - name: KEYCLOAK_API_ENVIRONMENT
                   value: ${KEYCLOAK_API_ENVIRONMENT}
+                # Request limits
+                # - name: MAX_REQ_BODY_SIZE
+                #   value: ${MAX_REQ_BODY_SIZE}
```
Comment (NickPhura) [inline api/.pipeline/templates/api.dc.yaml]: Commented out

File: api/.pipeline/templates/api.dc.yaml [hunk 2]
```diff
@@ -336,6 +346,13 @@ objects:
...
+                - name: MAX_UPLOAD_NUM_FILES
+                  value: MAX_UPLOAD_NUM_FILES
```
Comment (NickPhura) [inline api/.pipeline/templates/api.dc.yaml]: Probably needs to be wrapped in ${}

File: api/.pipeline/templates/api.dc.yaml (hunk 1 above)
Comment (MacDeluca) [reply api/.pipeline/templates/api.dc.yaml]: Intentional, just testing it fails when it's missing an env value

File: api/.pipeline/lib/api.deploy.js [hunk 3]
```diff
@@ -37,6 +37,7 @@ const apiDeploy = async (settings) => {
         // Persistent Volume
         VOLUME_CAPACITY: phases[phase].volumeCapacity,
         // BioHub Platform (aka: Backbone)
+        BACKBONE_PUBLIC_API_HOST: phases[phase].backbonePublicApiHost,
```
Comment (NickPhura) [inline api/.pipeline/lib/api.deploy.js]: Hmm, it seems like this is never actually used in the API anywhere. Maybe we should just remove it here, and remove it from the env-config schema

Comment (NickPhura) [reply api/.pipeline/lib/api.deploy.js]: Its passed into the compose.yml#api too, but I guess is never referenced after that

File: api/.pipeline/templates/api.dc.yaml (hunk 1 above)
Comment (NickPhura) [reply api/.pipeline/templates/api.dc.yaml]: Oh, unless you're testing it fails :)

File: api/.pipeline/templates/api.dc.yaml (hunk 2 above)
Comment (MacDeluca) [reply api/.pipeline/templates/api.dc.yaml]: Ooops my bad, nice catch

File: api/.pipeline/lib/api.deploy.js (hunk 3 above)
Comment (MacDeluca) [reply api/.pipeline/lib/api.deploy.js]: Yea doesn't look like it hey. Dropping.

File: api/.pipeline/templates/api.dc.yaml (hunk 1 above)
Comment (NickPhura) [reply api/.pipeline/templates/api.dc.yaml]: Oh weird, I thought I wrote my second message before you commented, but I guess I didnt refresh or something

**Review by NickPhura — APPROVED** [review]

**Pull request merged**